### ISSUE TITLE
Use of unresolved identifier 'MinLengthRule'

### ISSUE THREAD

**Issue by gberger** [jpotts18/SwiftValidator#13]
For some reason other rules work, but this one doesn't.

````
validator.registerField(emailField, rules: [RequiredRule(), EmailRule()])
validator.registerField(passwordField, rules: [RequiredRule(), MinLengthRule(length: 6)])
```

**Comment by gberger** [comment]
Also happens with RegexRule...

**Comment by jpotts18** [comment]
Are you using this with a cocoapod?

**Comment by jpotts18** [comment]
Looks like I needed to make the class public. Thanks for finding that!

**Comment by jpotts18** [comment]
Fixed in #16
